neuralprocesses/torch/nn.py

```diff
@@ -101,6 +114,10 @@ def ModuleList(modules):
     Conv2d = partial(ConvNd, dim=2)
     Conv3d = partial(ConvNd, dim=3)
 
+    UpSampling1d = partial(UpSamplingNd, dim=1)
```

Comment:
Does this work? `UpSamplingNd` seems to not take the keyword argument `dim`.

Reply:
Good catch, you're correct. PyTorch infers the dim from the shape and resizes automatically.

Reply:
Right! Can then simply remove the `dim=X`s I think. :)

Reply:
Done: https://github.com/wesselb/neuralprocesses/pull/1/commits/5963f3b60ffccf7b950a5501c74b7bda140b621c